app/assets/constants/buildings.rb

```diff
@@ -274,6 +273,76 @@ module Buildings
     ]
   ].freeze
 
+  HPI_LETTERS = [
+    {
+      coordinate: [52.39364, 13.1300582],
+      letter: 'I'
+    },
+    {
+      coordinate: [52.394252, 13.1319832],
+      letter: 'HS'
+    },
+    {
+      coordinate: [52.3934502, 13.1311611],
+      letter: 'A'
+    },
+    {
+      coordinate: [52.3936223, 13.1317085],
+      letter: 'B'
+    },
+    {
+      coordinate: [52.3934892, 13.132123],
+      letter: 'C'
+    },
+    {
+      coordinate: [52.3923204, 13.1235078],
+      letter: 'V'
+    },
+    {
+      coordinate: [52.3941746, 13.1330726],
+      letter: 'H'
+    },
+    {
+      coordinate: [52.3918618, 13.1237636],
+      letter: 'D'
+    },
+    {
+      coordinate: [52.3948646, 13.1245168],
+      letter: 'G'
+    },
+    {
+      coordinate: [52.3922193, 13.1239371],
+      letter: 'E'
+    },
+    {
+      coordinate: [52.3925689, 13.1239734],
+      letter: 'F'
+    },
+    {
+      coordinate: [52.3847794, 13.1190556],
+      letter: 'J'
+    },
+    {
+      coordinate: [52.3924289, 13.1248221],
+      letter: 'L'
+    }
+
+  ].freeze
+
+  def self.getLetters(hpi_letters)
+    div_icons = []
+    hpi_letters.each do |hpi_letter|
+      div_icons << {
+        latlng: hpi_letter[:coordinate],
+        div_icon: {
+          html: hpi_letter[:letter],
+          class_name: "building-icon"
+        }
+      }
+    end
+    div_icons
```

Comment:
This is `map` again, something like
```suggestion
    hpi_letters.map do |hpi_letter|
      {
        latlng: hpi_letter[:coordinate],
        div_icon: {
          html: hpi_letter[:letter],
          class_name: "building-icon"
        }
      }
    end
```

Reply:
this is now implemented